tests/http/test_handlers.py

```diff
@@ -93,6 +98,153 @@ def test_broadcast_to_client_without_ws_connection_present(self):
             client.ws_connection = None
         handlers.WebSocketHandler.broadcast("message", self.io_loop)
 
+    @tornado.testing.gen_test
+    def test_good_origin(self):
+        headers = {"Origin": "http://localhost", "Host": "localhost"}
+        conn = yield self.connection(headers=headers)
+        assert conn
+
+    @tornado.testing.gen_test
+    def test_bad_origin(self):
+        headers = {"Origin": "http://foobar", "Host": "localhost"}
+        with self.assertRaises(tornado.httpclient.HTTPClientError) as e:
+            _ = yield self.connection(headers=headers)
+        assert 403 == e.exception.code
```

Comment:
This looks like another yoda expression that should be flipped.

Reply:
Cheers! Fixed